calico-cloud/get-started/connect/install-automated.mdx

```diff
@@ -96,7 +96,7 @@ These commands can be added to any automated workflow.
     # clusterName is the unique name this cluster will have in Calico Cloud
     clusterName: my-cluster
     # calicoCloudVersion is the version to install
-    calicoCloudVersion: v19.1.0
+    calicoCloudVersion: {{cloudUserVersion}}
```

Comment:
Are you purposefully not adding this to the versioned version of this file?

Reply:
Nope, managed to missing pushing that change. Thanks for spotting this!